### DISCUSSION TITLE
How to enable prometheus on frontend-proxy on Kubernetes

### DISCUSSION THREAD

**Discussion by Vyshak1510** [open-telemetry/opentelemetry-demo#1365] [Q&A] [answered]
Hi Everyone, I'm trying to access prometheus publicly to scrape metrics from prometheus to an external tool. How do I do it?

**Comment by puckpuck** [comment]
The Prometheus service installed with the demo via Helm is used to receive all demo metrics via OTLP, and also scrape the Collector's self metrics to populate some dashboards.

If you need a Prometheus service to capture demo metrics, as well as scrape other services you do one of these 2 options:
1) Configure the Prometheus service installed with the Helm chart (see the values.yaml file for the chart)
2) Don't install the Prometheus service with the demo using `prometheus.enabled=false` and use your own Prometheus service instead. It will need to be configured to receive OTLP data, and scrape the OpenTelemetry Collector pod as well as any other scraping rules you want to specify.

**Reply by Vyshak1510** [reply]
Understood, thanks for the quick response. I have another question, if I have access the demo Prometheus, do I have to use kubectl port-forward svc/my-otel-demo-prometheus-server 9090:9090 ? I also want to run this application for quite some time on cloud, thinking of running it on minikube cuz it's easy

**Reply by puckpuck** [reply, marked as answer]
You can do a port-forward or setup the Prometheus sub chart to install with an externally exposed service/ingress.

**Reply by Vyshak1510** [reply]
Works, thank you
